cartoframes/context.py

```diff
@@ -148,6 +150,23 @@ def write(self, df, table_name, temp_dir='/tmp', overwrite=False,
         Returns:
             None
         """
+        if encode_geom:
+            is_geopandas = getattr(df, '_geometry_column_name')
```

Comment:
Sure, that's fine. I was using `_geometry_column_name` because it's [one of two things guaranteed by the `_metadata` of a geopandas dataframe](https://github.com/geopandas/geopandas/blob/master/geopandas/geodataframe.py#L41). Whichever string you want to check is fine with me!